libs/s25main/ingameWindows/IngameWindow.cpp

```diff
@@ -27,13 +28,15 @@ const DrawPoint IngameWindow::posAtMouse(std::numeric_limits<DrawPoint::ElementT
                                          std::numeric_limits<DrawPoint::ElementType>::max() - 1);
 
 const Extent IngameWindow::borderSize(1, 1);
+
+constexpr Extent IngameWindow::ButtonSize;
+
 IngameWindow::IngameWindow(unsigned id, const DrawPoint& pos, const Extent& size, std::string title,
                            glArchivItem_Bitmap* background, bool modal, CloseBehavior closeBehavior, Window* parent)
     : Window(parent, id, pos, size), title_(std::move(title)), background(background), lastMousePos(0, 0),
-      last_down(false), last_down2(false), isModal_(modal), closeme(false), isMinimized_(false), isMoving(false),
-      closeBehavior_(closeBehavior)
+      isModal_(modal), closeme(false), isMinimized_(false), isMoving(false), closeBehavior_(closeBehavior),
+      buttonStates_({ButtonState::Up, ButtonState::Up})
```

Comment:
I'd rather keep the `std::fill` below to avoid changing this when we add buttons